src/api/BioLink.js

```diff
@@ -113,9 +113,9 @@ const nodeAssociationTypes = {
     'disease',
     'gene',
     'phenotype',
-    'model',
-    'variant',
-    'literature'
+    // 'model',
```

Comment:
+1 to annotating thiese

Reply:
Biolink is missing genotype-to-model and genotype-to-variant associations right now. Not sure if genotype-to-literature will ever be supported.